cmd/reva/ocm-share-create.go

```diff
@@ -166,11 +166,11 @@ func ocmShareCreateCommand() *command {
 func getOCMSharePerm(p string) (*ocm.SharePermissions, int, error) {
 	if p == viewerPermission {
 		return &ocm.SharePermissions{
-			Permissions: conversions.NewViewerRole().CS3ResourcePermissions(),
+			Permissions: conversions.NewViewerRole(false).CS3ResourcePermissions(),
```

Comment:
can we use constants here for readability?

Reply:
Good point :+1: Fixed it